src/power_grid_model_io/converters/base_converter.py

```diff
@@ -7,7 +7,12 @@
 from abc import ABC, abstractmethod
 from typing import Generic, Optional, Tuple, TypeVar
 
+import logging
+logging.basicConfig(level=logging.ERROR)
+
 import structlog
+structlog.configure(wrapper_class=structlog.make_filtering_bound_logger(logging.ERROR))
```

Comment:
python good practice: import before doing anything else, 2 whitelines after final import

Reply:
Suggestions included in the newest commit.